matrix/Quaternion.hpp

```diff
@@ -4,11 +4,20 @@
  * All rotations and axis systems follow the right-hand rule.
  * The Hamilton quaternion product definition is used.
  *
- * In order to rotate a vector v by a righthand rotation defined by the quaternion q
+ * In order to rotate a vector in frame b (v_b) to frame n by a righthand
+ * rotation defined by the quaternion q_nb (from frame b to n)
  * one can use the following operation:
- * v_rotated = q^(-1) * [0;v] * q
- * where q^(-1) represents the inverse of the quaternion q.
- * The product z of two quaternions z = q1 * q2 represents an intrinsic rotation
+ * v_n = q_nb * [0;v_b] * q_nb^-1
+ *
+ * Just like DCM's: v_n = C_nb * v_b (vector rotation)
+ * M_n = C_nb * M_b * C_nb^(-1) (matrix rotation)
+ *
+ * or similarly
+ * v_b = q_nb^1 * [0;v_n] * q_nb
```

Comment:
A bit late to the party, but there is a `-` missing, should be `v_b = q_nb^-1 ...`.

Reply:
I agree, there's another ^(-1) missing. Can you quickly review https://github.com/PX4/Matrix/pull/48 please?

Reply:
Again late to the party, it's merged. Thanks for spotting 👍